spec/jobs/thumbnail_job_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+RSpec.describe ThumbnailJob do
+  let(:web_resource) { build(:edm_web_resource) }
+
+  it { is_expected.to be_processed_in :thumbnails }
+
+  context 'for edm_isShownBy' do
+    before do
+      web_resource.save # persist the edm_web_resource
```

Comment:
since the sidekiq queue adapter is running with Sidekiq::Testing.fake! at this point the enqueued jobs are never picked up so the thumbnails are only generated when subject.perform is called. So there's no need to skip the callbacks.

Since these are unit tests of the functionality that the job provides and not the functionality of whether or not a job is enqueued or the callbacks on an EDM::WebResource, this should be sufficient and also keeps the test overhead lighter as well as the test more modular.

I've added the checks for the files to `not_to exist` though, because they do make the test more thorough.